### PR TITLE
Backend endpoint recents

### LINKED ISSUE #7: BE Endpoint Recent Shouts
- [ ] /landing-page response:

- Date 
- Shouter Slack Handle
- Shoutee Slack Handle
- Message 
- Company Values

### PR DESCRIPTION
**Pull request by emackinnonTechtonic**
### Type of change:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling - no new features

### Why is this change required? What problem(s) does it solve?
On this PR I have broken out the routes and controllers into their own separate folders. The recent shouts endpoint is created to pull in the necessary data to the landing page's Carousel component.
### Were there any challenges while implementing this feature? If so, how were they addressed?
N/A
### Where should the reviewer start? How can this be tested?
We plan on making documentation to get the app running on a local machine.
### Link issues
#7

### CODE REVIEW

File: server/controllers/shoutoutController.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const environment = process.env.NODE_ENV || 'development';
+const config = require('../knexfile')[environment];
+const db = require('knex')(config);
```
Comment (tgeeTechtonic) [inline server/controllers/shoutoutController.js]: I think that these should probably stay in App and then you can just import `db` into the controller

File: server/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "knex": "^0.21.6",
+    "nodemon": "^2.0.6",
```
Comment (tgeeTechtonic) [inline server/package.json]: Should probably change this to a dev dependency and also add a dev start script as well

**Review by tgeeTechtonic — CHANGES_REQUESTED** [review]
I think we should reorganize a few things before merging this into develop, but otherwise, this looks great!

**Review by broxburyTechtonic — APPROVED** [review]

**Review by tgeeTechtonic — APPROVED** [review]
Looks good now!

**Pull request merged**